v0/destinations/freshmarketer/utils.js

```diff
@@ -46,36 +90,316 @@ const getUserAccountDetails = async (userEmail, Config) => {
       errorStatus
     );
   }
-  return userSalesAccountResponse;
+  let accountDetails =
+    userSalesAccountResponse.response.contact?.sales_accounts;
```

Comment:
```suggestion
    userSalesAccountResponse?.response?.contact?.sales_accounts;
```

Reply:
Do the same at other possible places as well

Reply:
okay